### DISCUSSION TITLE
Weather app

### DISCUSSION THREAD

**Discussion by BlackJack-14** [community/community#19679] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey sir can you please repost your react weather app code

**Comment by melvincwng** [comment, marked as answer]
@BlackJack-14 Hi Rudra, firstly, let me welcome you to GitHub Community👋

I saw your profile, and from your README.md - it seems that you are learning React.js. 

I happen to have previously built a [react weather app](https://github.com/melvincwng/react-weather-app) which you can use as reference. Is this what you are looking for?

Hope this helps you in your project.

Good luck, take care 😄 

If this reply has been helpful, don't forget to 'Mark as answer' or even follow me on GitHub.
 
Appreciate the support given in advance :)

**Reply by BlackJack-14** [reply]
thank you <3

**Reply by melvincwng** [reply]
@BlackJack-14, no problem :)
